### PR TITLE
FFC-175276: Recommend to pay confirmation

### PR DESCRIPTION
**Pull request by json-hamilton**
https://dev.azure.com/defragovuk/DEFRA-FFC/_sprints/taskboard/DEFRA-FFC%20Vet%20Visits%20Team/DEFRA-FFC/Vet%20Visits/Sprint%2030?workitem=175276

### CODE REVIEW

File: app/views/macros/recommend-form.njk [hunk 1]
```diff
@@ -5,7 +5,7 @@
       <div class="govuk-panel govuk-panel--confirmation govuk-panel--align-horizontal govuk-!-text-align-center">
         <div class="govuk-panel__body">
           <div class="govuk-grid-row">
-            <button class="govuk-button" onclick="location.href='';">Recommend to pay</button>
```
Comment (govTechSatish) [inline app/views/macros/recommend-form.njk:8]: Don't use js, you can use Gov Button and other Gov Frontend style. 
https://design-system.service.gov.uk/components/

File: app/views/macros/recommend-form.njk [hunk 2]
```diff
@@ -5,7 +5,7 @@
       <div class="govuk-panel govuk-panel--confirmation govuk-panel--align-horizontal govuk-!-text-align-center">
```
Comment (marcin-mogiela) [inline app/views/macros/recommend-form.njk:5]: should this be a confirmation panel?

File: app/routes/view-application.js [hunk 3]
```diff
@@ -62,7 +63,8 @@ module.exports = {
         payment: application?.payment,
         ...new ViewModel(application, applicationHistory),
         page: request.query.page,
-        recommendForm: rbacEnabled && displayRecommendationForm
+        recommendForm: rbacEnabled && displayRecommendationForm && !request.query.recommendToPay,
```
[the remark this answers is not in the document]
Comment (json-hamilton) [reply app/routes/view-application.js:66]: This query param is just a boolean that is being used to see if the recommend to pay button has been pressed, as once it is pressed the recommend buttons should hide and the confirmation screen should show, I don't think I can check if it has been clicked without this extra query param?

[the remark this answers is not in the document]
Comment (chrissmKainos) [reply app/routes/view-application.js:66]: Yeah, the query param I've used in claimhelper is doing exactly the same thing in that it is set on selection of the appropriate button on the initial screen. I think we are just ending up with the same query param being used in multiple areas and with different names.

request.query.approve
request.query.reject

**Review by chrissmKainos — APPROVED** [review]

**Review by marcin-mogiela — APPROVED** [review]

**Pull request merged**